project/plugins.sbt

```diff
@@ -7,3 +7,4 @@ addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "0.6.29")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh"                  % "0.3.7")
 addSbtPlugin("org.scoverage"      % "sbt-scoverage"            % "1.6.0")
 addSbtPlugin("net.virtual-void"   % "sbt-dependency-graph"     % "0.10.0-RC1")
+addSbtPlugin("com.eed3si9n"       % "sbt-assembly"             % "0.14.10")
```

Comment:
do we need this?

Reply:
I need it to produce the the CLI assembly. It doesn't work in `sbt` so I have to run it trough the Jar. That's still a problem, I couldn't find a way to make it progress in an sbt run. It hangs and needs to be killed. If started with
```
java -jar <the jar> localhost 6379
```
works fine.

Reply:
And yes, would love to understand why it hangs there if used within sbt